Makefile

```diff
@@ -37,15 +37,15 @@ else
 endif
 
 BIN_DIR = build/host/sdk/bin
-TOITPKG_BIN = $(BIN_DIR)/toitpkg$(EXE_SUFFIX)
-TOITLSP_BIN = $(BIN_DIR)/toitlsp$(EXE_SUFFIX)
-TOITVM_BIN = $(BIN_DIR)/toitvm$(EXE_SUFFIX)
-TOITC_BIN = $(BIN_DIR)/toitc$(EXE_SUFFIX)
+TOITPKG_BIN = $(BIN_DIR)/toit.pkg$(EXE_SUFFIX)
+TOITLSP_BIN = $(BIN_DIR)/toit.lsp$(EXE_SUFFIX)
+TOITVM_BIN = $(BIN_DIR)/toit.run$(EXE_SUFFIX)
+TOITC_BIN = $(BIN_DIR)/toit.compile$(EXE_SUFFIX)
 VERSION_FILE = build/host/sdk/VERSION
 CROSS_ARCH=
 
 # Note that the boot snapshot lives in the bin dir.
-TOIT_BOOT_SNAPSHOT = $(BIN_DIR)/toitvm_boot.snapshot
+TOIT_BOOT_SNAPSHOT = $(BIN_DIR)/run_boot.snapshot
```

Comment:
Can we drop _boot at some point? I think we call it system in most other places.

Reply:
Happy to change it.
Didn't know we called it something else.